### PR TITLE
fixed GRO file reading

### LINKED ISSUE #2276: Add loading .gro time series trajectory.
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by richardjgowers**
Fixes #2276

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/coordinates/GRO.py [hunk 1]
```diff
@@ -176,27 +176,29 @@ def _read_first_frame(self):
...
-            grofile.seek(0)
-            for pos, line in enumerate(grofile, start=-2):
+            for pos, line in enumerate(grofile, start=1):
                 # 2 header lines, 1 box line at end
                 if pos == n_atoms:
                     unitcell = np.float32(line.split())
-                    continue
-                if pos < 0:
-                    continue
+                    break
```
Comment (richardjgowers) [inline package/MDAnalysis/coordinates/GRO.py]: this is the real change, just break when you hit the end of data..

**Comment by richardjgowers** [comment]
@MDAnalysis/coredevs this is good to go too

**Comment by richardjgowers** [comment]
@orbeckst done

**Comment by orbeckst** [comment]
Please merge. I can't do it from mobile. Thanks @richardjgowers !

**Pull request merged**